Kiss.Bff.EndToEndTest/Vraag/VraagScenarios.cs

```diff
@@ -367,6 +367,248 @@ public async Task vragenValueValidation()
 
         }
 
+        [TestMethod("5. Save klantcontact with many characters in onderwerp")]
+        public async Task When_SavingContactmomentWithLongTitleAndEmptyVraag_ExpectOnderwerpToBeTruncated()
```

Comment:
I don't want to be verry picky but where does 'Title' refer to in the testname? and what do you mean by 'emptyVraag'? isn't the Vac in the vraag?
How about
When_SavingContactmomentWithLongVraag_Expect_VraagToBeTruncatedInTheContactmomentOnderwerp